src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -145,46 +149,56 @@ define(function (require, exports, module) {
             addGutter(currentEditor);
             handleDocumentChange();
         }
-    }
+	}
```

Comment:
This is wrong.  Take a look at this:

![screen shot 2017-04-21 at 3 13 46 pm](https://cloud.githubusercontent.com/assets/427398/25292737/6b8665b2-26a5-11e7-88cd-f0b2edd2a274.png)

See how the increased indent you've added means that the function no longer lines up with its closing `}`?